### PR TITLE
Fix script for sending link to expo app to discussion related to PR

### PR DESCRIPTION
**Pull request by jaroslav-kubicek**
---
Please check this before merging (do not delete this checklist):

- [ ] it works in landscape and portrait mode
- [ ] it uses `cacheConfig.force=true` if offline access doesn't make sense

### CODE REVIEW

File: scripts/expo/commands/publishCodeReview.js [hunk 1]
```diff
@@ -48,14 +48,16 @@ module.exports = () => {
     }),
   });
 
-  const githubPullRequestId = env.CI_PULL_REQUEST_ID;
```
Comment (mrtnzlml) [inline scripts/expo/commands/publishCodeReview.js]: This is not OK - you need to use values from this file: https://github.com/kiwicom/react-native-app/blob/master/scripts/expo/environment.js (`env.*` is not `process.env` here).

**Comment by mrtnzlml** [comment]
@jaroslav-kubicek Seems like it's really broken, but I would recommend to ignore it because we are removing Expo...

File: scripts/expo/commands/publishCodeReview.js (hunk 1 above)
Comment (jaroslav-kubicek) [reply scripts/expo/commands/publishCodeReview.js]: I know, just trying... 🙂

**Comment by jaroslav-kubicek** [comment]
💔

**Pull request closed without merging**